templates/cli/lib/commands/pull.js.twig

```diff
@@ -108,10 +108,30 @@ const pullFunctions = async ({ code, withVariables }) => {
             if (allowCodePull) {
                 log("Pulling active deployment's code ...");
 
+                let deploymentId = func['deployment'];
+
+                try {
+                    const fetchResponse = await functionsListDeployments({
+                        functionId: func['$id'],
+                        queries: [
+                            JSON.stringify({ method: 'limit', values: [1] }),
+                            JSON.stringify({ method: 'orderDesc', values: [''] })
+                        ],
+                        parseOutput: false
+                    });
+
+                    if (fetchResponse['total'] > 0) {
+                        deploymentId = fetchResponse['deployments'][0]['$id'];
+                    }
+
+                } catch (e) {
+                    log("Can't get last deployment ID, using local value instead")
+                }
```

Comment:
I dont think we need fallback. If there is no deployment in listDeployments, the active deployment will also never work